src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -295,7 +296,7 @@ private void testAddAction() throws Exception {
 
         newSession.setFeedbackSessionName("Allow Early Viewing Session #");
         newSession.setCourseId("CFeedbackUiT.CS1101");
-        newSession.setTimeZone(-4.5);
+        newSession.setTimeZone(TimeHelper.convertToZoneId(-4.5));
```

Comment:
Just use something like `ZoneId.of("-04:30")`? `convertToZoneId` should be removed eventually.